PIPReader/src/test/java/it/cnr/iit/ucs/properties/UCFPipProperties.java

```diff
@@ -9,18 +9,18 @@
 
 public class UCFPipProperties implements PipProperties {
 
-    private String className;
+    private String name;
```

Comment:
may be attributeName?

Reply:
no this is the current property name of any property.
It can be either a className or packageName

Reply:
for example:

Reply:
I see

Reply:
ok but also class/className is fine I'll change it again